lib/utils.pm

```diff
@@ -1128,8 +1128,7 @@ sub reconnect_mgmt_console {
             }
         }
 
-        # SLE >= 15 does not offer auto-started VNC server in SUT, only login prompt as in textmode
-        if (!check_var('DESKTOP', 'textmode') && is_sle('<15')) {
+        if (!check_var('DESKTOP', 'textmode')) {
```

Comment:
Looks like we don't use that in QAM tests, but do you know if that patch is available in SLE 15 SP0 update channels?

Reply:
Indeed I don't think QAM ues this, I'm checking just in case.

Reply:
Found only QR test which could be affected but it's [textmode](https://openqa.suse.de/tests/3284860), rest of tests are custom.
Should be fine.

Reply:
Thanks a lot!

Reply:
JFYI, it's also available in SLE 15 SP0 update channels.